### ISSUE TITLE
Still work on this project?

### ISSUE THREAD

**Issue by zhengrenwen** [ur7hfo/simple-automatic-antenna-tuner-on-stm32f103#1]

**Comment by ur7hfo** [comment]
no. the project is over. I'm currently working on another project.

**Comment by zhengrenwen** [comment]
I am actually doing one thing like this .ATU for loop antenna from 7-14mhz with simple LC network.mcu is 103ZET6.but not good at programing.

and my request is follow:
1.kind of antenna have you work with your atu?
2.how about the tuing time?and success rate？
3.refer to another n7ddc atu project with source code,im stuck in the tuing alogrithm to control relay .i want to copy to my new mcu ,can provide some ideas or guides?

---Original---
From: "dima ***@***.***&gt;
Date: Tue, May 17, 2022 00:34 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [ur7hfo/simple-automatic-antenna-tuner-on-stm32f103] Still workon this project? (Issue #1)




 
no. the project is over. I'm currently working on another project.

**Comment by zhengrenwen** [comment]
by the way, appreciate for your reply 😃



---Original---
From: "dima ***@***.***&gt;
Date: Tue, May 17, 2022 00:34 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [ur7hfo/simple-automatic-antenna-tuner-on-stm32f103] Still workon this project? (Issue #1)




 
no. the project is over. I'm currently working on another project.

**Comment by achanger-huang** [comment]
I also want to refer to your STM32 project source code, can you share, thank you very much.

**Comment by achanger-huang** [comment]
> by the way, appreciate for your reply 😃
> […](#)
> ---Original--- From: "dima ***@***.***&gt; Date: Tue, May 17, 2022 00:34 AM To: ***@***.***&gt;; Cc: ***@***.******@***.***&gt;; Subject: Re: [ur7hfo/simple-automatic-antenna-tuner-on-stm32f103] Still workon this project? (Issue #1) no. the project is over. I'm currently working on another project. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

Hello, zhengrenwen, I am also solving the same problem, can I add your contact information to learn and communicate?
